src/Microsoft.ComponentDetection.Contracts/TypedComponent/LinuxComponent.cs

```diff
@@ -9,12 +9,14 @@ private LinuxComponent()
         /* Reserved for deserialization */
     }
 
-    public LinuxComponent(string distribution, string release, string name, string version)
+    public LinuxComponent(string distribution, string release, string name, string version, string license = null, string author = null)
```

Comment:
Can these be nullable? `string?`. Same for the properties below.

Reply:
They can be but the analyzer kept complaining. This was the only way I could get it to not show errors. I didn't want to use "#nullable enable" for the entire file.